orbit/pkg/table/extension_darwin.go

```diff
@@ -44,6 +45,13 @@ func PlatformTables() []osquery.OsqueryPlugin {
 		table.NewPlugin("corestorage_logical_volumes", corestorage.LogicalVolumesColumns(), corestorage.LogicalVolumesGenerate),
 		table.NewPlugin("corestorage_logical_volume_families", corestorage.LogicalVolumeFamiliesColumns(), corestorage.LogicalVolumeFamiliesGenerate),
 
+		// Copied Kolide tables
+		user_exec.TablePlugin("user_screenlock", osquerydPath, "SELECT enabled, grace_period FROM screenlock",
+			[]table.ColumnDefinition{
```

Comment:
missing `user`?

Reply:
@lucasmrod where do you think we miss "user" ?

Reply:
@lucasmrod Can you explain what you are thinking here?